rapyuta_io_sdk_v2/client.py

```diff
@@ -1127,6 +1143,12 @@ def commit_revision(
     ) -> Munch:
         """Commit a revision.
 
+        Args:
+            name (str): Config tree name
+            revision_id (str): Config tree revision ID
+            configTreeRevision (object): Config tree revision details
```

Comment:
This should be `config_tree_revision`. 
```suggestion
            configTreeRevision (object): Config tree revision details
```